rollup.config.js

```diff
@@ -19,7 +19,8 @@ module.exports = [
         indent: false,
         plugins: [],
         globals: {
-          'chart.js': 'Chart'
+          'chart.js': 'Chart',
+          'chart.js/helpers': 'Chartjs Helpers'
```

Comment:
This value represents the property under which [the helpers object is accessible](https://github.com/chartjs/Chart.js/blob/master/src/index.js#L29) when importing Chart.js via script. I'm not sure if it works with nested property but I guess it should be `'chart.js/helpers': 'Chart.helpers'`.

Reply:
Same as https://github.com/chartjs/chartjs-plugin-datalabels/blob/master/rollup.config.js#L24